install/kubernetes/tetragon/values.yaml

```diff
@@ -200,6 +200,8 @@ tetragon:
     # "oci-hooks" (https://github.com/containers/common/blob/main/pkg/hooks/docs/oci-hooks.5.md).
     interface: "oci-hooks"
     installDir: "/opt/tetragon"
+    # -- Comma-separated list of namespaces to allow Pod creation for, in case tetragon-oci-hook fails to reach Tetragon agent.
+    failAllowNamespaces: ""
```

Comment:
Can we document that if this is left empty, the namespace where tetragon is installed will be used by default?

Reply:
Solved via https://github.com/cilium/tetragon/pull/2404/commits/2424f81efbc2ef85b5301390c08eb9afb87048db